### PR TITLE
[WIP]Fix Windows build break and delete unused yaml files

### PR DESCRIPTION
**Pull request by snnn**
**Description**: 
Fix Windows build break and delete unused yaml files

**Motivation and Context**
- Why is this change required? What problem does it solve?
Currently, Windows builds are failing with:

"pip : WARNING: You are using pip version 20.0.2; however, version 20.1 is available."

It's just a warning. However, it was output to stderr, then it was treated as error by the Azure pipeline's task.

- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: tools/ci_build/github/azure-pipelines/azure-pipelines-py-packaging.yml [hunk 1]
```diff
@@ -178,7 +178,9 @@ jobs:
       workingFolder: '$(Build.BinariesDirectory)'
 
   - script: |
-     python -m pip install -q pyopenssl setuptools wheel numpy==1.16.6
+     python -m pip install --upgrade pip
+     python -m pip uninstall -y -q onnxruntime 2>pip.log
```
Comment (edgchen1) [inline tools/ci_build/github/azure-pipelines/azure-pipelines-py-packaging.yml:182]: will pip.log ever be read? looks like everything will be cleaned up later by clean-agent-build-directory-step.yml. if not, how about just redirecting to /dev/null?

also, have you considered using script's failOnStderr setting?

Comment (snnn) [reply tools/ci_build/github/azure-pipelines/azure-pipelines-py-packaging.yml:182]: But failOnStderr is default false, so it confused me.

This is Windows, I can't use /dev/null.

Comment (davidbrownellWork) [reply tools/ci_build/github/azure-pipelines/azure-pipelines-py-packaging.yml:182]: Windows supports `command > null 2>&1` if you want to go that route.

**Comment by snnn** [comment]
Doesn't work. Will reopen it when I find a solution.

**Pull request closed without merging**